apps/workers/db-check/main.go

```diff
@@ -12,23 +12,22 @@ import (
 )
 
 const (
-	taskType    = "task.dbcheck"
+	taskType    = "task.db-check"
 	description = `
-Runs record count checks against a given table's data to verify that the data is updated as expected
+Runs checks against a given table's data to verify that the data is updated as expected
 
 uri params:
-db        - required database name to use for query connections
-table     - required table name to validate
-dt_column - required column name that holds the date value to query
-date      - the date to compare data against tolerance level
-offset    - if given, this is the offset duration back from date to compare tolerance
-tolerance - (default 0.05) percent is the allowed percentage of deviation between offset and compare row counts
-              0.05 (5%) means the difference between the row counts can vary by 5 percent and be accepted
-              a greater difference would send an alert, if compare is not provided this check is not done
-
-Example task 
-(table.schema checked for records where my_date = 2022-01-02 and 2022-01-01 has 10% as many records as 2022-01-02):
-  {"type":"task.dbcheck","info":"?db=mydbname&table=table.schema&dt_column=my_date&date=2022-01-02&offset=24h&tolerance=0.1"}`
+db_srv     - database server/source connection
+table      - schema.table name to validate
+type       - type of check
```

Comment:
Would you add a list of the support types here and a description of how they work